src/components/book-card/z-book-card/styles.css

```diff
@@ -4,189 +4,235 @@
-                color: var(--red500);
-              }
-            }
-          }
-        }
-      }
-    }
-
-    ::slotted([slot="apps"]) {
-      border-top: solid var(--border-size-small) var(--color-surface03);
-    }
-  }
-}
-
-/* Desktop breakpoint */
-
-/* 
-@media (min-width: 1152px) {
-} */
-
-/* Wide breakpoint */
-
-/* @media (min-width: 1366px) {
-} */
+:host > article .main-content {
+  display: flex;
+  flex-direction: column;
+  gap: calc(var(--space-unit) * 2);
+}
+
+:host > article .main-content .cover {
+  display: flex;
+  overflow: hidden;
+  height: 378px;
+  margin: 0 var(--space-unit);
+  border-radius: var(--border-radius);
+}
+
+:host > article .main-content .cover img {
+  max-width: 100%;
+  max-height: 100%;
+  align-self: end;
+}
+
+:host > article .main-content .card-info {
+  display: flex;
+  flex: 1;
+  flex-direction: column;
+  justify-content: space-between;
+  gap: calc(var(--space-unit) * 2);
+}
+
+:host > article .main-content .card-info .book-data {
+  display: flex;
+  flex: 1;
+  flex-direction: column;
+  justify-content: space-between;
+  margin: 0 var(--space-unit);
+  gap: var(--space-unit);
+}
+
+:host > article .main-content .card-info .book-data .authors-title-cta-section {
+  display: grid;
+  column-gap: calc(var(--space-unit) * 2);
+  grid-template-areas:
+    "authors authors"
+    "title cta";
+  grid-template-columns: auto 24px;
+}
+
+:host > article .main-content .card-info .book-data .authors-title-cta-section .authors {
+  display: -webkit-box;
+  overflow: hidden;
+  height: 20px;
+  margin-bottom: calc(var(--space-unit) / 2);
+  -webkit-box-orient: vertical;
+  grid-area: authors;
+  -webkit-line-clamp: 1;
+  line-clamp: 1;
+  text-overflow: ellipsis;
+}
+
+:host > article .main-content .card-info .book-data .authors-title-cta-section .opera-title {
+  display: -webkit-box;
+  overflow: hidden;
+  height: 54px;
+  -webkit-box-orient: vertical;
+  grid-area: title;
+  -webkit-line-clamp: 2;
+  line-clamp: 2;
+  text-overflow: ellipsis;
+}
+
+:host > article .main-content .card-info .book-data .authors-title-cta-section .opera-title * {
+  all: unset;
+}
+
+:host > article .main-content .card-info .book-data .isbn-tags-link-section .isbn-tags-section {
+  margin-bottom: var(--space-unit);
+}
+
+:host > article .main-content .card-info .book-data .isbn-tags-link-section .isbn-tags-section .volume-title {
+  display: -webkit-box;
+  overflow: hidden;
+  height: 20px;
+  -webkit-box-orient: vertical;
+  -webkit-line-clamp: 1;
+  line-clamp: 1;
+  text-overflow: ellipsis;
+}
+
+:host > article .main-content .card-info .book-data .isbn-tags-link-section .isbn-tags-section .isbn {
+  display: -webkit-box;
+  overflow: hidden;
+  height: 20px;
+  -webkit-box-orient: vertical;
+  -webkit-line-clamp: 1;
+  line-clamp: 1;
+  text-overflow: ellipsis;
+}
+
+:host > article .main-content .card-info .book-data .isbn-tags-link-section .isbn-tags-section .tags {
+  display: flex;
+  overflow: hidden;
```

Comment:
in questo modo ci perdiamo una porzione del focus del primo tag

Reply:
Già, però ci serve per tagliare eventuali righe successive alla prima, se i tag dovessero andare a capo. Devo capire se c'è una soluzione.

Reply:
se è una pigna lasca stare, lo teniamo così com'è

Reply:
Con l'ultimo commit dovrei aver risolto.